cpp/include/cudf/strings/combine.hpp

```diff
@@ -127,25 +129,25 @@ std::unique_ptr<column> join_strings(
  * @throw cudf::logic_error if the number of rows from @p separators and @p strings_columns
  *                          do not match
  *
- * @param strings_columns List of strings columns to concatenate.
+ * @param strings_columns List of strings columns to concatenate
  * @param separators Strings column that provides the separator for a given row
- * @param separator_narep String that should be used in place of a null separator for a given
- *        row. Default of invalid-scalar means no row separator value replacements.
- *        Default is an invalid string.
- * @param col_narep String that should be used in place of any null strings
- *        found in any column. Default of invalid-scalar means no null column value replacements.
- *        Default is an invalid string.
+ * @param separator_narep String to replace a null separator for a given row.
```

Comment:
There are inconsistencies for periods in param comments all through here. In some places, you are removing them, but in most places, you are using them. Should we be consistent with these?

Reply:
This has come up many times before. We should use periods only for sentences. 
I will not be able to make everyone happy so I'm going with Mark's recommendation here: https://github.com/rapidsai/cudf/pull/14247#discussion_r1349788189

Reply:
If I am to be an owner of some new repository, I will write an official dev guide from the beginning: "All @params must be followed by complete sentence(s) that begin with a capitalized letter and end with period" :smile:

Reply:
Fair enough. We have thought about it and are following a plan. I am fine with this.